lib/trento/application/projectors/check_result_projector.ex

```diff
@@ -157,21 +169,32 @@ defmodule Trento.CheckResultProjector do
         _,
         _
       ) do
-    TrentoWeb.Endpoint.broadcast("monitoring:clusters", "checks_results_updated", %{
-      cluster_id: cluster_id,
-      host_id: host_id,
-      hosts_executions: [
-        %{cluster_id: cluster_id, host_id: host_id, reachable: reachable, msg: msg}
-      ],
-      checks_results:
-        Enum.map(checks_results, fn %{check_id: check_id, result: result} ->
-          %{host_id: host_id, check_id: check_id, result: result}
-        end)
-    })
+    TrentoWeb.Endpoint.broadcast(
+      "monitoring:clusters",
+      "checks_results_updated",
+      ChecksResultView.render("checks_results_updated.json",
+        data: %{
+          cluster_id: cluster_id,
+          host_id: host_id,
+          reachable: reachable,
+          msg: msg,
+          checks_results:
```

Comment:
We could even move this `checks_results` rendering to the view. Maybe sending a new field like `on_request: true` to set the result to `unknown` or use the actual result otherwise.
But maybe it is too much XD

Reply:
this projector is going to go away so I would not spend a lot of time trying to refactor it